code/modules/item_worth/worths_list.dm

```diff
@@ -474,7 +474,7 @@ var/list/worths = list(
 					/obj/item/clothing/suit/space/void/security = 3000,
 					/obj/item/clothing/suit/space/void = 2800,
```

Comment:
A normal voidsuit has a value of 2800.
Is there a reason why the skrell suit is cheaper ?

Reply:
That is just how it was before I came to touch it, all I did was change the pathing. Ask the person that put that value to it if you want an answer to that.